examples/coap/src/bin/coapclient.rs

```diff
@@ -53,7 +53,7 @@ fn client_handshake() -> Result<(), EDHOCError> {
 
     let mut msg_3 = Vec::from(c_r.as_cbor());
     let (mut initiator, message_3, prk_out) =
-        initiator.prepare_message_3(CredentialTransfer::ByReference, &None)?;
+        initiator.prepare_message_3(CredentialTransfer::ByValue, &None)?;
```

Comment:
Maybe this and the `+ 32` below were pulled in by mistake from a previous commit?

Reply:
Kind of -- I hadn't pulled before working from your branch, now following the rebase.